env/Dockerfile.texttosql

```diff
@@ -1,25 +1,36 @@
-# This vLLM Dockerfile is used to construct image that can build and run vLLM on x86 CPU platform.
-# Based on https://github.com/PZD-CHINA/vllm/blob/55e1b7b54e916a3a930fc92fadca8a61adf9ace7/Dockerfile.cpu
+# This Dockerfile is used to construct image that can build and run texttosql model on x86 CPU platform.
+# Based on https://github.com/bigPYJ1151/vllm/blob/384623538c081ed621b04c1eec107132920e5045/Dockerfile.cpu
 
 FROM ubuntu:22.04
 
+# http and https proxy setup
+ENV http_proxy=http://child-prc.intel.com:913
+ENV https_proxy=http://child-prc.intel.com:913
+
 RUN apt-get update  -y \
     && apt-get install -y git wget vim numactl gcc-12 g++-12 python3 python3-pip \
     && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 10 --slave /usr/bin/g++ g++ /usr/bin/g++-12
 
-RUN pip install --upgrade pip
-RUN pip install --proxy http://child-prc.intel.com:913 wheel packaging ninja setuptools>=49.4.0 numpy
-RUN pip install --proxy http://child-prc.intel.com:913 torch==2.1.2+cpu --index-url https://download.pytorch.org/whl/cpu
+RUN pip install --upgrade pip \
+    && pip install wheel packaging ninja setuptools>=49.4.0 numpy
+    
+# git proxy setup
+RUN git config --global http.proxy http://child-prc.intel.com:913
 
-RUN pip install langchain --proxy http://child-prc.intel.com:913
+RUN git clone -b cpu_pr https://github.com/bigPYJ1151/vllm.git /workspace/vllm
 
-RUN pip install --proxy http://child-prc.intel.com:913 pyspark pyspark-ai
+WORKDIR /workspace/vllm
 
-RUN git config --global http.proxy http://child-prc.intel.com:913
-RUN git clone -b vllm-cpu-only https://github.com/PZD-CHINA/vllm.git /workspace/vllm
+RUN pip install -v -r requirements-cpu.txt --extra-index-url https://download.pytorch.org/whl/cpu
 
-WORKDIR /workspace/vllm
+# install langchain
+RUN pip install langchain
 
-RUN VLLM_BUILD_CPU_ONLY=1 MAX_JOBS=8 pip install --proxy=http://child-prc.intel.com:913 --no-build-isolation  -v -e .
+# install pyspark pyspark-ai
+RUN pip install pyspark
 
+RUN git clone https://github.com/PZD-CHINA/pyspark-ai.git /workspace/pyspark-ai
```

Comment:
Change to use oap-project/pyspark-ai url and create a pr for that repo to hold your code changes.